<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButtonReplaceAll">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>全部替换(&amp;A)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>查找规则</string>
   </property>
   <widget class="QCheckBox" name="checkBoxReverse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>反向查找</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWhole">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全词匹配</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>匹配大小写</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxReg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>正则表达式</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSure">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="labelCount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>计数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>22</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>查找目标(&amp;T):</string>
        </property>
        <property name="buddy">
         <cstring>lineEditSearch</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="text">
         <string>查找下一个</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>替换目标(&amp;R):</string>
        </property>
        <property name="buddy">
         <cstring>lineEditReplace</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditReplace"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReplace">
        <property name="text">
         <string>替换当前词</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
